src/emu/epoc/src/services/akn/skin/chunk_maintainer.cpp

```diff
@@ -80,6 +81,16 @@ namespace eka2l1::epoc {
         std::uint32_t param_type_;
     };
 
+    struct skn_scalable_item_def {
```

Comment:
Sorry again :p Can you follow the naming of upper structs. Like akns_srv_scaleable_item_def. It just seems weird that scaleable item is left alone!

Reply:
sorry, changes are pushed, didn't notice it when it moving the struct